internal/buffer/message.go

```diff
@@ -20,7 +21,8 @@ const (
 )
 
 // PrepareType represents a subtype for prepare messages.
-//go:generate stringer -type=PrepareType
+//
+//go:generate source ../../.bingo/variables.env && ${STRINGER} -type=PrepareType
```

Comment:
I am not a huge fan of loading the environment variables like this inside the go generate. Is it possible to simply use the go path where all dependencies are installed?

Reply:
Kindly correct me if i am wrong here, as of now i don't see the `String()` method being generated do we even need this `go:generate` line?

Reply:
with bingo the catch i see for this is that the binaries are versioned, so you would have to refer it like say `stringer-1.0.1` in the go files and each time you are upgrading the version you have to come and change here. That is the hazzle that `source ../../.bingo/variables.env` avoids